hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -100,8 +99,9 @@
   private final long lockCheckMinWaitTime;
   private final long lockCheckMaxWaitTime;
   private final FileIO fileIO;
+  private final CachedClientPool metaClients;
 
-  protected HiveTableOperations(Configuration conf, HiveClientPool metaClients, FileIO fileIO,
+  protected HiveTableOperations(Configuration conf, CachedClientPool metaClients, FileIO fileIO,
```

Comment:
What is the least restrictive parameter that we can use here? `HiveClientPool`, or `ClientPool`?

Reply:
I changed it to ClientPool